### DISCUSSION TITLE
CM Version

### DISCUSSION THREAD

**Discussion by filip11111** [quietvoid/dovi_tool#94] [Q&A] [answered]
When I generate DV from HDR10+, should I use CMv2.9 or CMv4.0?

Also, is it necessary to include "target_nits"?

Thank you very much!

**Comment by quietvoid** [comment, marked as answer]
The version doesn't really matter unless you have a recent display which supports it.
It's backwards compatible.

You don't need to specify `target_nits`, it makes no difference.
